app-emulation/vmware-addons/vmware_addons-1.2.2.recipe

```diff
@@ -4,32 +4,35 @@ between Haiku, running in a virtual machine, and the host operating system. \
 It is like the official VMware tools on Windows or Linux.
 
 It currently provides:
- * Clipboard sharing between Haiku and the host OS.
+ * Clipboard sharing between Haiku and the host OS. Use the VMware add-ons \
+desktop applet to enable or disable the sharing.
  * Mouse sharing: the mouse can seamlessly enter and leave the VM window.
  * Disk compacting: starts the VMware \"shrink\" process, which reduces the \
 size of \"auto-expanding\" virtual disks attached to the virtual machine. \
 The free space on disks is cleaned up previously, in order to get better \
 results.
  * Graphics driver for VMware: you can choose your preferred resolution \
-using Screen preflet in Haiku.
- * Graphics driver for VirtualBox: change grapchics card in VM settings \
-to VMSVGA and video memory size to 64 MB to get higher resolutions working.
- * Shared folders: folders from the host OS can be mounted in Haiku:
-   mount -t vmwfs ~/Shared"
+using the Screen preflet in Haiku.
+ * Graphics driver for VirtualBox: change the graphics controller in the \
+VM settings to VMSVGA, and set the video memory size to 64 MB to be able \
+to use higher resolutions.
+ * Shared folders: folders from the host OS can be mounted in Haiku: \
+mount -t vmwfs ~/Shared
+   "
 
 HOMEPAGE="https://github.com/HaikuArchives/VMwareAddons"
 COPYRIGHT="1999 Be, Inc.
 	2007 Haiku, Inc.
 	2009 Vincent Duvert"
 LICENSE="MIT"
-REVISION="1"
-srcGitRev="7781fdc08c327709801054afa1c30709b9bcae60"
+REVISION="2"
+srcGitRev="0ad9c858e914dae1b784cfdccf20f7ba458c0a74"
 SOURCE_URI="https://github.com/HaikuArchives/VMwareAddons/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="d55e2c32cc3f42e43c3c4b333d8d25b438bab3ce87d0a9a6e96c2d0ac0f559a5"
+CHECKSUM_SHA256="2b4869301c53d94a3f74bdc119e1e9caeb5ba94f5f389f840b4232e879cd09eb"
 SOURCE_FILENAME="VMwareAddons-$srcGitRev.tar.gz"
 SOURCE_DIR="VMwareAddons-$srcGitRev"
 
-ARCHITECTURES="all"
+ARCHITECTURES="x86_64 x86_gcc2"
```

Comment:
VMware addons only works on `x86_64` and `x86_gcc2` as of now, and won't work on another architecture without additional adaption, because of `x86` assembly usage for the VMware backdoor access.

See also https://github.com/haikuports/haikuports/pull/6152#discussion_r710215491